org.opent2t.sample.thermostat.superpopular/com.nest.thermostat/js/thingTranslator.js

```diff
@@ -23,13 +24,17 @@ function findResource(schema, di, resourceId) {
         return d.di === di; 
     }); 
     
-    if (!entity) throw new Error('NotFound');
+    if (!entity) {
+        throw new Error('NotFound');
```

Comment:
Being fixed by Dan's changes

---
In reply to: [103830547](https://github.com/openT2T/translators/pull/188#discussion_r103830547) [](ancestors = 103830547)